core-bundle/src/Resources/contao/widgets/RootPageDependentModule.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao;
+
+/**
+ * @property array $rootPages
+ */
+class RootPageDependentModule extends SelectMenu
```

Comment:
Can you explain why we need a special widget?

Reply:
How would you render this thing with multiple selects?

Reply:
Let me rephrase: I can see why you need a special widget here but why is this extending `SelectMenu` then? Or in other words: You add a special widget which is only here for `rootPages`. That makes no sense to me. Why not allow a `MultiSelectMenu`. You already load lots of stuff with callbacks anyway so maybe it can be generalized? Right now it's a mix.

Reply:
I see, I like the idea. 👍

Reply:
After our disscusion on Slack I went for `RootPageDependentSelect` as a generalized solution would be more in the direction of a MCW / group widget. Would you prefer `RootPageSelect` instead?

Reply:
`RootPageDependentSelect` 👍